routing/pathfind.go

```diff
@@ -310,13 +310,6 @@ func newRoute(amtToSend, feeLimit lnwire.MilliSatoshi, sourceVertex Vertex,
 		return nil, err
 	}
 
-	// Invalidate this route if its total fees exceed our fee limit.
```

Comment:
Do we have tests to cover this case for the regular path finding? Took at quick skim through the tests and couldn't' find anything equivalent.

Reply:
Yes, there is one: https://github.com/lightningnetwork/lnd/blob/cfb5e249b9d6dae9e44dcf56c29c3eda531290fe/routing/pathfind_test.go#L709